masterlist.yaml

```diff
@@ -6237,3 +6237,14 @@ plugins:
     clean:
       - crc: 0xB0CE38E4
         util: 'FO4Edit v4.0.3h (Hotfix 1)'
+        
+  - name: 'Skip New Game Intro.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/28562' ]
+    dirty:
+      - <<: *quickClean
+        crc: 0xA99E0D9B
+        util: '[FO4Edit v4.0.4b](https://www.nexusmods.com/fallout4/mods/2737)'
+        itm: 3
+    clean:
+      - crc: 0x6092FFBA
+        util: 'FO4Edit v4.0.4b'
```

Comment:
We exclude cleaning data for dirty plugins after they've been cleaned due to volatility in their checksums.

```suggestion
```